shim/src/commons.js

```diff
@@ -0,0 +1,32 @@
+// Define a library of shorthands. Sharing these declarations
+// accross modules improves both consitency and minification.
+// Unused declarations are dropped by the tree shaking process.
+
+const defProp = Reflect.defineProperty;
+const deleteProp = Reflect.deleteProperty;
+const getProto = Reflect.getPrototypeOf;
+const setProto = Reflect.setPrototypeOf;
+
+const getOwnPropDesc = Object.getOwnPropertyDescriptor;
+const getOwnPropDescs = Object.getOwnPropertyDescriptors;
+const getOwnPropNames = Object.getOwnPropertyNames;
+
+const ownKeys = Reflect.ownKeys;
+
+const create = Object.create;
+const assign = Object.create;
```

Comment:
Thanks David. I am breaking-down my changes in smaller commits, and this one got out of sync. Good suggestion.